browser/src/docdispatcher.ts

```diff
@@ -332,6 +332,10 @@ class Dispatcher {
 				behavior: 'smooth',
 			}); // Scroll up based on stylepreview height
 		};
+
+		this.actionsMap['gotopage'] = () => {
+			app.map.sendUnoCommand('.uno:GotoPage');
```

Comment:
I think it doesn't make sense to add new dispatch command to call uno command.
dispatch commands are for actions which require browser-side code.
Please instead call sendUnoCommand in the target place